extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -443,15 +536,15 @@ public function getStatsData( $startYear, $startMonth, $startDay, $endYear, $end
 		$startDate = $startYear . '-' . $startMonth . '-' . $startDay . ' 00:00:00';
 		$endDate = $endYear . '-' . $endMonth . '-' . $endDay . ' 23:59:59';
 
-		$summary = array(
+		$summary = [
 			'all' => 0,
 			ImageReviewStatuses::STATE_APPROVED 	=> 0,
 			ImageReviewStatuses::STATE_REJECTED 	=> 0,
 			ImageReviewStatuses::STATE_QUESTIONABLE => 0,
 			'avg' => 0,
-		);
-		$data = array();
-		$userCount = $total = $avg = 0;
+		 ];
+		$data = [];
+		$total = $avg = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$avg" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)